src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1091,6 +1091,37 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
         }
 
         [OuterLoop] // TODO: Issue #11345
+        [Theory]
+        [InlineData("")] // missing size
+        [InlineData("10000000000000000")] // overflowing size
+        public async Task GetAsync_InvalidChunkSize_ThrowsHttpRequestException(string chunkSize)
+        {
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                using (var client = new HttpClient())
+                {
+                    string partialResponse = "HTTP/1.1 200 OK\r\n" +
+                        "Transfer-Encoding: chunked\r\n" +
```

Comment:
Ok.  I just copied this from the above test, which doesn't have one either:
https://github.com/dotnet/corefx/pull/23160/files/82a127bd4700f6c73c051d05c8ea3e95eed1a5db#diff-06a6483ff6bb478e06f199d69522bcd4L1070
I'll leave it like this for now and we can subsequently scrub through them to add the header where it's needed.